### DISCUSSION TITLE
Unit test crash when using 15.5 sim

### DISCUSSION THREAD

**Discussion by rudro** [pointfreeco/swift-perception#11] [General]
### Description

I'm on macOS 14.2 and Xcode 15.0.  I tried to run the unit tests on 17.x and everything was great, when I realized I was just getting the fall through code path.

So I switched to a iPhone 13 Pro iOS 15.5 sim.  And I see the following crash:
<img width="1405" alt="Screenshot 2024-01-11 at 1 22 56 PM" src="https://github.com/pointfreeco/swift-perception/assets/126961/f9a12555-ea44-44cd-a99c-83e36f079463">

I then tried with iOS 16.4 and the unit tests seem to work out fine.

### Checklist

- [X] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-perception/issues) or [discussion](https://github.com/pointfreeco/swift-perception/discussions).

### Expected behavior

No crashes.

### Actual behavior

Crash.

### Xcode version information

15.2

**Comment by mbrandonw** [comment]
This is just a crash in test code that doesn't have to do with the library (something to do with `ImageRenderer`), so I am going to convert this to a discussion. I don't have easy access to iOS 15 right now, but if you figure out a workaround we will happily take a PR to fix.
